commit 606d137afc1afc2ab5e9c4949ce1c78bec1848fe
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Sat May 15 07:57:55 2021 +0800

    Fix compatibility with tasty-hspec 1.1.7
    
    tasty-hspec no longer re-exports Test.Hspec since 1.1.7. Let's import
    it ourselves.

diff --git a/dhall-lsp-server/dhall-lsp-server.cabal b/dhall-lsp-server/dhall-lsp-server.cabal
index 9c407256..2e23e615 100644
--- a/dhall-lsp-server/dhall-lsp-server.cabal
+++ b/dhall-lsp-server/dhall-lsp-server.cabal
@@ -104,6 +104,7 @@ Test-Suite tests
     Build-Depends:
         base                                   ,
         haskell-lsp-types >= 0.19.0  && < 0.25 ,
+        hspec             >= 2.7     && < 2.9  ,
         lsp-test          >= 0.9     && < 0.13 ,
         tasty             >= 0.11.2  && < 1.5  ,
         tasty-hspec       >= 1.1     && < 1.2  ,
diff --git a/dhall-lsp-server/tests/Main.hs b/dhall-lsp-server/tests/Main.hs
index 401bc0b2..5d6b548f 100644
--- a/dhall-lsp-server/tests/Main.hs
+++ b/dhall-lsp-server/tests/Main.hs
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP                   #-}
 {-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE OverloadedStrings     #-}
 
@@ -17,6 +18,10 @@ import Language.Haskell.LSP.Types
 import Test.Tasty
 import Test.Tasty.Hspec
 
+#if MIN_VERSION_tasty_hspec(1,1,7)
+import Test.Hspec
+#endif
+
 import qualified Data.Text       as T
 import qualified GHC.IO.Encoding
 
